### PR TITLE
Some Bugfixes

### PR DESCRIPTION
**Pull request by EagleEyeBW**
add headers field
fix body for not get/post requests
fix name attribute of post request

### CODE REVIEW

File: ajax-form.html [hunk 1]
```diff
@@ -242,8 +243,8 @@
                             this._params[field.name || field.id] = value;
                         }
 
-                        else if (_this.method.toLowerCase() === 'post') {
-                            this._body[field.id || field.id] = value;
+                        else /*if (_this.method.toLowerCase() === 'post' || _this.method.toLowerCase() === 'put')*/ {
```
Comment (kyzrfranz) [inline ajax-form.html:246]: why did you add new checks and then remove the checking altogether?

Comment (EagleEyeBW) [reply ajax-form.html:246]: Because, I uses PUT but than I remember that more HTTP Methods uses Parameters

Comment (kyzrfranz) [reply ajax-form.html:246]: ^^ - I'll add and merge - everybody feel free to chip in ..

**Review by kyzrfranz — APPROVED** [review]
